src/types.ts

```diff
@@ -60,7 +62,15 @@ export interface NodeFileTraceResult {
   warnings: Error[];
 }
 
-export default function NodeFileTrace(
-  files: string[],
-  opts: NodeFileTraceOptions
-): Promise<NodeFileTraceResult>;
+export interface SingleValue {
+  value: string | string[] | undefined;
+  wildcards?: string[];
+}
```

Comment:
I'm not sure if the `wildcards` option should be part of `SingleValue` or if I should make a new `WildcardValue` type?

Reply:
This does seem simpler to me... maybe just call it `StaticValue` or something like that?